pkg/rulesConfig/defaultRules/10-ensure-sca-scanner.go

```diff
@@ -40,14 +50,6 @@ func githubErrorsRule10(githubData map[string]*githubConnector.GithubOwner) ([]*
 		".*snyk/actions/maven@.*",
 	}
 
-	runRegexExpressions := []string{
-		".*^[\\S]*trivy.*|.*docker .* run .*(aquasec/)?trivy.*",
-		"^[\\S]*grype|docker .* run .*(anchore/)?grype.*",
-		"(jfrog|jf) (s|scan).*",
-		"ws scan.*",
-		"snyk (code | )test.*",
-	}
-
 	for _, owner := range githubData {
 		for _, repo := range owner.Repositories {
 			foundScaScanner := false
```

Comment:
What's the reason Gitlab has a separate function to test SCA, while Github's implementation is all in one function?

Reply:
gitlab came second so I put a bit more thought into it